projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -314,15 +316,20 @@ const ProductDetailCard = ( {
 					) ) }
 				</ul>
 
-				{ needsPurchase && discountPrice && (
+				{ needsPurchase && productPrice && (
 					<>
 						<div className={ styles[ 'price-container' ] }>
-							<Price value={ discountPrice } currency={ currencyCode } isOld={ false } />
-							{ discountPrice < price && (
+							<Price value={ productPrice } currency={ currencyCode } isOld={ false } />
+							{ productPrice < price && (
 								<Price value={ price } currency={ currencyCode } isOld={ true } />
 							) }
 						</div>
 						<Text className={ styles[ 'price-description' ] }>{ priceDescription }</Text>
+						{ introductoryOffer?.reason && (
+							<Text className={ styles[ 'price-note' ] }>
+								{ __( '* Not eligible for introductory discount', 'jetpack-my-jetpack' ) }
```

Comment:
Do we want to show this note/message?
Or should we just show the un-discounted full price without a note/message?
Any thoughts or opinion about this?